ci/axis/cpu.yml

```diff
@@ -1,9 +1,27 @@
+# Copyright (c) 2018-2020 NVIDIA Corporation
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Released under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+
+SDK_TYPE:
+  - nvcr.io/nvidia/cuda
```

Comment:
Can these be `cuda` and `nvhpc`? I don't have a way in Jenkins to split this out before the docker image is pulled.

The `cccl` side can be left as is. It's just the thrust side and how Jenkins will pull the docker image that's an issue.

Reply:
Ah yah I can do that, no problem. I'll change them.

Reply:
Does it help it make the change in `cccl` as well?

Reply:
The consistency is probably worth it

Reply:
Fixed here.

Reply:
Done in `cccl` as of https://github.com/NVIDIA/cccl/commit/d6134ad5cdfd5aa37c5209906c5acf0fdebe7d3b